packages/app/src/components/theme/TextStyle.tsx

```diff
@@ -0,0 +1,76 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { memo, useMemo } from 'react';
+import styled from 'styled-components';
+import { sketchColorToRgbaString, sketchColorToRgba } from 'noya-designsystem';
+
+// It has very basic bacground color preview.
+export type SimpleTextDecoration = 'none' | 'underline' | 'strikethrough';
```

Comment:
We should only have one copy of this, but I'll fix it after we merge. I'm going to rearrange a few other things too